packages/nx/src/command-line/init/configure-plugins.ts

```diff
@@ -64,19 +59,16 @@ export async function installPlugin(
     });
     return;
   }
-  const { implementationFactory } = getGeneratorInformation(
-    plugin,
-    initGenerator,
-    repoRoot,
-    {}
+  execSync(
+    `${pmc.exec} nx g ${plugin}:init --keepExistingVersions ${
+      updatePackageScripts ? '--updatePackageScripts' : ''
+    } ${verbose ? '--verbose' : ''}`,
+    {
+      stdio: [0, 1, 2],
+      cwd: repoRoot,
+      windowsHide: false,
+    }
```

Comment:
This should match closer to the older implementation here:
https://github.com/nrwl/nx/blob/68db3e032cfe24c66b44a230033fe3032536ba53/packages/nx/src/command-line/add/add.ts#L121-L162

* use the init generator that was found
* try/catch the exec

Reply:
actually, i added it back it. i did not remove above code

Reply:
Oh I see, sorry I missed it because it was collapsed 😅